src/java.base/share/classes/java/security/DigestInputStream.java

```diff
@@ -51,6 +51,15 @@
  * retain a handle onto the digest object, and clone it for each
  * digest to be computed, leaving the original digest untouched.
  *
+ * @implNote This implementation updates and only updates the message digest
+ *      with data actually read from the input stream when it is turned on.
+ *      This includes various {@code read} methods, {@code transferTo},
+ *      {@code readAllBytes}, and {@code readNBytes}. Specifically, data
```

Comment:
Specifically
->
Additionally ?

Reply:
I meant additionally as in "here's an additional case to be aware of...", not "this is an additional read case..."